content/en/docs/installation/updating.md

```diff
@@ -77,7 +77,7 @@ It may help to correctly load some configurations and/or daemons.
 
 ---
 
-# Maintainer section
+## Maintainer section
```

Comment:
why does the linter complain here? can I not make a h2 after a h1 heading?

Reply:
It is the other way around. You are not allowed to add a h1 heading after a h2 heading.

https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md025---multiple-top-level-headers-in-the-same-document